src/extlibs/hrvo/agent.h

```diff
@@ -5,10 +5,10 @@
 #include "software/ai/navigator/path_planner/hrvo/velocity_obstacle.h"
 #include "software/geom/vector.h"
 
-class HRVOSimulator;
+class HrvoSimulator;
 
 /**
- * An agent/robot in the HRVO simulation.
+ * An agent/robot in the Hrvo simulation.
```

Comment:
I think acronyms can be left in all-caps in the comments? This also applies to other changes to comments. 
```suggestion
 * An agent/robot in the HRVO simulation.
```